EIPS/eip-7266.md

```diff
@@ -0,0 +1,46 @@
+---
+eip: 7266
+title: Remove blake2f (0x09) Precompile
+description: Remove the blake2f (0x09) precompile by changing the precompile behaviour to be a dataless revert
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/discussion-removal-of-ripemd-160-and-blake2f-precompiles/14857
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-07-03
+---
+
+## Abstract
+
+This EIP removes the [`blake2f`](./eip-152.md) (`0x09`) precompile by changing the precompile behaviour to be a dataless (`0x`) revert.
+
+## Motivation
+
+[EIP-152](./eip-152.md) has never capitalised on a real-world use case. This fact is clearly reflected in the number of times the address `0x09` has been invoked (numbers from the date this EIP was created):
+
+- The most recent call took place on 6 October 2022.
```

Comment:
```suggestion
- The most recent call took place on 6 October, 2022.
```

Reply:
Since I stick consistently to the British English language, I disagree with this recommendation.

Reply:
:+1: :england: